### PR TITLE
Fix flake8 + pyramid test class name

### PR DESCRIPTION
**Pull request by petrus-v**

### CODE REVIEW

File: {{cookiecutter.project_slug}}/{{cookiecutter.python_package}}/{{cookiecutter.blok_name}}/tests/test_pyramid.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from anyblok_pyramid.tests.testcase import PyramidBlokTestCase
 
 
-class TestCanigooRestApi(PyramidBlokTestCase):
```
Comment (jssuzanne) [inline {{cookiecutter.project_slug}}/{{cookiecutter.python_package}}/{{cookiecutter.blok_name}}/tests/test_pyramid.py:4]: I am wondering, who write the example :D

**Review by jssuzanne — APPROVED** [review]

File: {{cookiecutter.project_slug}}/{{cookiecutter.python_package}}/{{cookiecutter.blok_name}}/tests/test_pyramid.py (hunk 1 above)
Comment (franckbret) [reply {{cookiecutter.project_slug}}/{{cookiecutter.python_package}}/{{cookiecutter.blok_name}}/tests/test_pyramid.py:4]: Me too ;-)

**Comment by petrus-v** [comment]
I've merged my branch as @jssuzanne approved it few hours ago !

**Comment by petrus-v** [comment]
And thanks for review !

**Pull request merged**